.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -268,8 +272,8 @@ jobs:
                 libxkbcommon-dev
             workspace: /home/runner/cxx-qt
 
-          - name: macOS 12 (clang) Qt5
-            os: macos-12
+          - name: macOS 14 (clang) Qt5
+            os: macos-14-large
```

Comment:
i don't think we should be using the `-large` ones are they are part of the paid github side rather than the "standard" free runners?